Translations/ko-KR/update.json

```diff
@@ -3,7 +3,7 @@
   "equipment_type": "1.3.83",
   "expedition": "20071603",
   "operation": "20071534",
-  "quest": "21030469",
+  "quest": "21030470",
   "ship": "21030173",
```

Comment:
The ship version should be raised since the ship file was changed.

Reply:
Solved!